website/addons/dataverse/model.py

```diff
@@ -40,12 +43,30 @@ def provider(self):
 
     @property
     def version_identifier(self):
-        return 'state'
+        return 'version'
 
     @property
     def unique_identifier(self):
         return self.file_id
 
+    def enrich(self, save=True):
+        super(DataverseFile, self).enrich(save)
+
+        # Get user object from cookie
+        cookie = request.cookies.get(settings.COOKIE_NAME)
+        session_id = itsdangerous.Signer(settings.SECRET_KEY).unsign(cookie)
+        session = Session.load(session_id) or Session(_id=session_id)
+        user = User.load(session.data['auth_user_id'])
```

Comment:
It looks like I can.

When testing this, is it fine to patch the return value to be the current user, or is there a better way?